client/my-sites/checkout/composite-checkout/lib/generic-redirect-processor.ts

```diff
@@ -43,31 +42,21 @@ export default async function genericRedirectProcessor(
 		throw new Error( 'Required purchase data is missing' );
 	}
 
-	const { protocol, hostname, port, pathname } = parseUrl(
```

Comment:
Do you know why `parseUrl` was being used here?

Reply:
Not really. My guess is that it was added when folks didn't know how else to get the url parts from the url.